app/src/main/java/co/develhope/chooseyouowncocktail_g2/domain/DrinkMapper.kt

```diff
@@ -47,12 +48,19 @@ object DrinkMapper {
             strCategory,
             ingrList,
             strDrinkThumb,
-            false
+            false,
+            incSortId()
         )
     }
 
     fun listToDomainModel(dtoList: DrinksResult): List<Drink> {
         return dtoList.drinks.map { it.mapToDomainModel() }
```

Comment:
visto che bisogna mappare una lista su una lista, puoi spostare il map all'intern odella funzione di mapping così è più pulito

Reply:
Se ti riferisci alla funzione listToDomainModel(), non appartiene a questo branch. 
Forse è meglio fare la modifica in una pr separata?